docarray/array/mixins/find.py

```diff
@@ -106,6 +107,8 @@ def find(
         :param exclude_self: if set, Documents in results with same ``id`` as the query values will not be
                         considered as matches. This is only applied when the input query is Document or DocumentArray.
         :param only_id: if set, then returning matches will only contain ``id``
+        :param index:  if the query is a string, text search will be performed on `index`, otherwise, ignored. By
```

Comment:
This docstring does not help me understand what this parameter is about or does

Reply:
updated